### PR TITLE
Fix visibility issue of MediaController while navigation and scrolling

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
https://dimagi-dev.atlassian.net/browse/MOB-117
**Problem**: Since The MediaController will create a default set of controls and put them in a window floating above your application., the mediacontroller will not respect parent view's animation or scroll.

**Solution**: Add the mediaController to the view hierarchy to force it always conforms parent view's animation and scroll.

**Extra**: Prevent logging inline video events if the video is never played.

PS: A duplicate of this https://github.com/dimagi/commcare-android/pull/2168, since Jenkins builds gets really messy and produces some weird logs for failure even though there isn't one. 

Product Note: UI improvements for Inline Video

### CODE REVIEW

File: app/src/org/commcare/views/media/CommCareVideoView.java [hunk 1]
```diff
@@ -39,6 +39,10 @@ public void start() {
     @Override
     protected void onDetachedFromWindow() {
         super.onDetachedFromWindow();
+        // Do not log events if the video is never played.
+        if (duration == 0) {
```
Comment (shubham1g5) [inline app/src/org/commcare/views/media/CommCareVideoView.java]: Can we always fire the listener and do this check in the listener implementation. Would also be good to separate this out from this PR.

Comment (ShivamPokhriyal) [reply app/src/org/commcare/views/media/CommCareVideoView.java]: Separated out this change here https://github.com/dimagi/commcare-android/pull/2179

**Comment by ShivamPokhriyal** [comment]
@damagatchi retest this please

**Comment by ShivamPokhriyal** [comment]
@damagatchi retest this please.

**Pull request merged**